lib/htmltoword/document.rb

```diff
@@ -61,10 +66,21 @@ def generate
               out.write(source)
             elsif @replaceable_files[entry.name]
               out.write(@replaceable_files[entry.name])
+            elsif entry.name == Document.content_types_xml_file
```

Comment:
I've changed this with my latest commit. We now read the contents of the content_types file and inject the required tags based on the image extensions included in the document.